### ISSUE TITLE
Plotting functions for outputs and states

### ISSUE THREAD

**Issue by jmgnve** [jmgnve/VannModels.jl#12]
Should this be included in this package, or in external package? Or even something more general? A shiny app?

**Comment by kolbjornengeland** [comment]
I think it is useful to have one very simple plot included in this package and make an additional package later on, either Julia or as a shiny app. Or maybe it is suffucient only to give one example of how to plot in the Readme file?

**Comment by jmgnve** [comment]
Yes, we can give one example in the readme for now. Then, when we have a good way of exporting states we can look at this further.

**Comment by ElinLangsholt** [comment]
To ensure a short way to the Flomvarslingsapp, the export format should match with the suggested Flomvarslingsapp standard input format.
